src/transport_ofi.h

```diff
@@ -354,6 +354,42 @@ shmem_transport_put_small(void *target, const void *source, size_t len, int pe)
 
 static inline
 void
+shmem_transport_put_large(void *target, const void *source, size_t len, int pe)
```

Comment:
Could you change the name to something like ```shmem_transport_ofi_put_large``` so that it's clear this isn't part of the transport layer interface?